README.md

```diff
@@ -16,51 +16,19 @@ Paintera is a general visualization tool for 3D volumetric data and proof-readin
 
 ## Installation and Usage
 
-Paintera is available for installation through [conda](#conda) and the [Python Package Index](#pip). Alternatively, you can use [jgo](#jgo) to run Paintera from an existing Maven artifact on the [SciJava Maven repository](https://maven.scijava.org) or your local maven repository, typically located at `$HOME/.m2` on your computer. For all of these options, Java 8 with JavaFX and Apache Maven are requirement. Efforts to create a standalone app have not been successful so far ([#253](https://github.com/saalfeldlab/paintera/issues/253)).
+Paintera is available for installation through [conda](#conda) and the [Python Package Index](#pip). Alternatively, you can use [jgo](#jgo) to run Paintera from an existing Maven artifact on the [SciJava Maven repository](https://maven.scijava.org) or your local maven repository, typically located at `$HOME/.m2` on your computer. For all of these options, Java 11 and Apache Maven are required. Efforts to create a standalone app have not been successful so far ([#253](https://github.com/saalfeldlab/paintera/issues/253)).
```

Comment:
the text says java 11 in many places, but we need java 13, correct?

Reply:
It's set to target Java 11 currently, though it is set to compile with 13. Initially I did this since I figured it would be easiest to rely on `conda` for the jdk installation, since they now have `openjdk 11.0.6`. However, looking more into this now, it seems that the linux version still installs java 8 (https://github.com/conda-forge/openjdk-feedstock/issues/57), so we likely still need to depend on a system install for the JDK. I can update these to indicate Java 13

Reply:
I misspoke in the above comment. There is an `openjdk` conda package which has a similar issue, but seemingly the `conda-forge` version no longer does. When using `openjdk` through conda-forge, we are able to grab [Java 11 (conda-forge)](https://anaconda.org/conda-forge/openjdk).